Lunch/CreateAccountViewController.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  CreateAccountViewController.swift
+//  Lunch
+//
+//  Created by Spencer Amann on 8/1/15.
+//  Copyright (c) 2015 Spencer Amann. All rights reserved.
+//
+
+import UIKit
+import Parse
+
+class CreateAccountViewController: UIViewController {
+
+    @IBOutlet weak var userNameTextField: UITextField!
+    @IBOutlet weak var emailTextField: UITextField!
+    @IBOutlet weak var passwordTextField: UITextField!
+    @IBOutlet weak var confirmedPasswordTextField: UITextField!
+
+    let loginIdentifier = "loginAfterCreateAccount"
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    @IBAction func createAccountButtonClick(sender: AnyObject) {
```

Comment:
I'd call this `createAccountButtonTapped` and use a more specific type for the parameter, like `UIButton` in this case